src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -127,15 +130,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String name = request.getParameter("course-name");
     String prof = request.getParameter("prof-name");
     Long units = Long.parseLong(request.getParameter("num-units"));
-    String term = request.getParameter("term-name");
+    String term = request.getParameter("term");
+    String school = request.getParameter("school-name");
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Entity newCourse = new Entity("Course");
+
+    AddSchoolData addSchool = new AddSchoolData();
+    addSchool.addSchoolData(datastore, request);
```

Comment:
This might give odd behavior, not sure what instantiating a servlet inside a servlet will do.

Keep that in mind if you have issues, might want to extract the majority of the functionality to a data class. 

Fine for the demo though.